doc/release/release_0.20.rst

```diff
@@ -0,0 +1,642 @@
+- Fix typo in docstring of ``skimage.measure.moments_hu``
+  (`#6016 <https://github.com/scikit-image/scikit-image/pull/6016>`_).
+- Fix formatting of mode parameter in ``skimage.util.random_noise``
+  (`#6532 <https://github.com/scikit-image/scikit-image/pull/6532>`_).
+- Fix broken links in SKIP 3
+  (`#6445 <https://github.com/scikit-image/scikit-image/pull/6445>`_).
+- Fix broken link in docstring of ``skimage.filters.sobel``
+  (`#6474 <https://github.com/scikit-image/scikit-image/pull/6474>`_).
+- Change "neighbour" to EN-US spelling "neighbor"
+  (`#6204 <https://github.com/scikit-image/scikit-image/pull/6204>`_).
+- Add missing copyrights to LICENSE.txt and use formatting according to SPDX identifiers
+  (`#6419 <https://github.com/scikit-image/scikit-image/pull/6419>`_).
+- Include ``skimage.morphology.footprint_from_sequence`` in the public API documentation
+  (`#6555 <https://github.com/scikit-image/scikit-image/pull/6555>`_).
+- Correct note about return type in the docstring of ``skimage.exposure.rescale_intensity``
+  (`#6582 <https://github.com/scikit-image/scikit-image/pull/6582>`_).
+- Stop using the ``git://`` connection protocol and remove references to it
+  (`#6201 <https://github.com/scikit-image/scikit-image/pull/6201>`_, `#6283 <https://github.com/scikit-image/scikit-image/pull/6283>`_).
+- Update scikit-image's mailing addresses to the new domain discuss.scientific-python.org
+  (`#6255 <https://github.com/scikit-image/scikit-image/pull/6255>`_).
+- Remove references to deprecated mailing list in ``doc/source/user_guide/getting_help.rst``
+  (`#6575 <https://github.com/scikit-image/scikit-image/pull/6575>`_).
+- Use "center" in favor of "centre", and "color" in favor of "colour" gallery examples
+  (`#6421 <https://github.com/scikit-image/scikit-image/pull/6421>`_, `#6422 <https://github.com/scikit-image/scikit-image/pull/6422>`_).
+- Replace reference to ``api_changes.rst`` with ``release_dev.rst``
+  (`#6495 <https://github.com/scikit-image/scikit-image/pull/6495>`_).
+- Clarify header pointing to notes for latest version released
+  (`#6508 <https://github.com/scikit-image/scikit-image/pull/6508>`_).
+- Add missing spaces to error message in ``skimage.measure.regionprops``
+  (`#6545 <https://github.com/scikit-image/scikit-image/pull/6545>`_).
+- Apply codespell to fix common spelling mistakes
+  (`#6537 <https://github.com/scikit-image/scikit-image/pull/6537>`_).
+- Add missing space in math directive in normalized_mutual_information's docstring
+  (`#6549 <https://github.com/scikit-image/scikit-image/pull/6549>`_).
+- Fix lengths of docstring heading underline in ``skimage.morphology.isotropic_`` functions
+  (`#6628 <https://github.com/scikit-image/scikit-image/pull/6628>`_).
+- Fix plot order due to duplicate examples with the file name ``plot_thresholding.py``
+  (`#6644 <https://github.com/scikit-image/scikit-image/pull/6644>`_).
+- Get rid of numpy dep warning in gallery example ``plot_equalize``
```

Comment:
```suggestion
- Get rid of numpy deprecation warning in gallery example ``plot_equalize``
```
To me "dep" means "dependency" :wink:

Reply:
I'm confused. You say it means "dependency" but you suggest "deprecation"? :sweat_smile: I'm guessing deprecation is the one you mean?

Reply:
Nope: Here, "dep" means "deprecation" (what would a "dependency warning" would be, btw? :wink:) -- so let's spell it out. Plus, let's avoid any confusion, because 'usually' "dep" is used to mean "dependency" -- at least in my dictionary.